content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -262,9 +262,11 @@ In the above two cases of the `Guaranteed` pod, the `none` CPU Manager policy wo
 
 In the case of the `BestEffort` pod, the `static` CPU Manager policy would send back the default topology hint as there is no CPU request and the Device Manager would send back the hints for each of the requested devices.
 
-Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments. 
+Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments.
+
+### NUMA Topology-aware Scheduling
+The [out-of-tree scheduler](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/noderesourcetopology) now could mitigate problem when kube-scheduler performs sub-optimal node assignments, without taking into account the behaviour of ToplogyManager on the worker node. To work properly this plugin should be subscribed to NodeResourceTopology modification generated by Node Resource Topology Exporter, whichis now part of [Node Feature Discovery] (https://github.com/kubernetes-sigs/node-feature-discovery).

```

Comment:
There are a couple of typos here:

s/ToplogyManager /Topology Manager
s/whichis/which is

I think this can be phrased a bit better. We can move some the text in the know limitation section and use that to capture the motivation, something on the lines of:

```
The default scheduler is not topology-aware, so it is possible to be scheduled on a node and then fail on the node due to the Topology Manager. However, [out-of-tree scheduler](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/noderesourcetopology) scheduler plugin mitigates this problem and allows optimal scheduling by taking resource topology into consideration.
In order to achieve this, the plugin needs to subscribe to NodeResourceTopology modification generated by Node Resource Topology Exporter which exposes resource information with NUMA node granularity.  An implementation of such an exporter is [Node Feature Discovery] (https://github.com/kubernetes-sigs/node-feature-discovery).
```

WDYT?

Reply:
ok, I agree to put it back to the "Known Limitations" as you suggested.